install.sh

```diff
@@ -34,7 +34,7 @@ nvm_profile_is_bash_or_zsh() {
   local TEST_PROFILE
   TEST_PROFILE="${1-}"
   case "${TEST_PROFILE-}" in
-    *"/.bashrc" | *"/.bash_profile" | *"/.zshrc")
+    *"/.bashrc" | *"/.bash_profile" | *"/.zshrc" | "/etc/bashrc")
```

Comment:
Hi ljharb, for example in a context of a Vagrant deployment: as we act as root during provision command, it is easy to deploy it like this if I want to use my company domain credentials instead of the basic vagrant user. I understand the per-user vision of nvm of course, but it is quite sad to restrict it like this :)

Reply:
Sure but in that environment you don’t need the install script - you can set it up manually in your vagrantfile.

Reply:
Of course I can install it manually, but there is an install script... Anyway, I don't insist. Maybe something you can put in your roadmap is to **ensure** the user is not providing a "system" PROFILE file, because it leads to bad behavior during installation. Last question before closing this thread: Why this tool is per-user only ? What prevent the tool to run correctly if it is installed in a global location ? :)

Reply:
Because if it’s global, then non-root users will be forced to try to use sudo to install global packages or alter installed node versions, and one of the primary benefits of nvm is that you never ever need sudo once installing it.

Reply:
Is it only a question of permission ? Because I obviously got it that nvm configuration would be shared across multiple non-root users. But in my case, we are dealing with personal virtual machines creation only used by one user. And in my example, I deployed it inside a non-root writable directory `/opt/nvm` so no `sudo` needed. Is there an undefined `nvm` behavior in a global configuration context ?

Reply:
No, with all those caveats you'll be fine - in that case you're using it for a single user, it's just that that user happens to be root (altho, it's a better idea to have your main user in a vagrant/docker image *not* be root).

Given all the constraints though, I don't think it's a good idea to account for that in common code paths.

Reply:
Users are not root, just authorized to write things in the nvm directory ;) OK, let's cancel that Pull Request. But to me there is something to fix somewhere because the message shown during the installation: `=> bash_completion source string already in /etc/bashrc` (which is wrong) should not appear, or something should prevent to handle other than user PFOFILE files.

Reply:
I totally agree that seems wrong; maybe you could update this PR to improve the message?

Reply:
Actually, the vision of keeping nvm per user is great, because some peoples has no sudo acces on their computers. I'm using nvm in my work computer and developers has not priviliges to use sudo in my company and no rights to write things to root directory for security purposes.